src/validation/overseas.entity.query.validation.ts

```diff
@@ -4,7 +4,7 @@ import { ErrorMessages } from "./error.messages";
 import { VALID_OE_NUMBER_FORMAT } from "./regex/regex.validation";
 
 export const overseasEntityQuery = [
-  body("oe_number")
+  body("entity_number")
     .not().isEmpty({ ignore_whitespace: true })
     .withMessage(ErrorMessages.OE_QUERY_NUMBER)
     .matches(VALID_OE_NUMBER_FORMAT)
```

Comment:
Should we change the naming of OE_QUERY_NUMBER and VALID_OE_NUMBER_FORMAT as well?

Reply:
No. There will be design input coming. This is just to allow cost to flow through to api.